northstar/src/runtime/console.rs

```diff
@@ -397,7 +410,15 @@ where
 
             // If the connections breaks: just break. If the receiver is dropped: just break.
             let mut take = ReaderStream::with_capacity(stream.get_mut().take(size), 1024 * 1024);
-            while let Some(buf) = take.next().await {
+            while let Some(buf) = timeout(NPK_STREAM_TIMEOUT, take.next())
+                .await
+                .map_err(|_| {
+                    Error::Io(
+                        "timeout".into(),
+                        io::Error::new(io::ErrorKind::TimedOut, "timeout"),
+                    )
+                })?
```

Comment:
nit: How about using our `Context` trait clone instead of a map to an IO error? `.context("NPK stream timed out")?`